package-lock.json

```diff
@@ -5422,7 +5422,7 @@
         "spdx-expression-parse": "^4.0.0"
       },
       "engines": {
-        "node": ">=18"
+        "node": ">=20.11.0"
```

Comment:
I think this is ~ok because `20` is the minimum supported, but I'm not sure on the full versioning (or why this change didn't come with the `package.json` engine min change.

Reply:
this is the same in https://github.com/IBM/cloudant-node-sdk/pull/1803/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519L5425

Reply:
I wonder if that comes from one of the dev deps then 
🤷 
it doesn't make any practical difference anyway